themes/10up-theme/assets/css/styleguide/styleguide.css

```diff
@@ -138,7 +137,7 @@
 
 .uikit__chevron-up,
 .uikit__chevron-down {
-	fill: currentColor;
+	fill: currentcolor;
```

Comment:
@joesnellpdx is this preferred in lowercase? I see it documented on a few sites as camelCase. Just wanted to be sure.

Reply:
@devinle I changed because prettier was erroring out on this and didn't want it to be a blocker.

Reply:
@devinle test this when you run it in the AM. It really should be camel case (from my recollection) - but I don't want to wrangle with prettier for the sake of our styleguide styles right now.